docs/src/http_api_guide.md

```
@@ -0,0 +1,411 @@
+
+| Name         | Type   | Description              |
+| ------------ | ------ | ------------------------ |
+| header_hash | string | Hash of the block header |
+
+#### Description
+Returns all of the UTXOs included in the block identified by the provided hash.
+
+> Note: This can be a significant number of UTXOs with their accompanying metadata. Please use cautiously.
+
+#### Response
+| Name                                 | Type             | Description                                       |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)